CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+### 0.5.1 (unreleased)
+
+- New features
+  - More consistent handling of missing keys: if a config key is missing pureconfig always throws a
+    `KeyNotFoundException` now, unless the `ConfigConvert` extends the new `AllowMissingKey` trait.
+
+### 0.5.0 (Jan 3, 2017)
+
+- New features
+  - Sealed families are now converted to and from configs unambiguously by using an extra `type` field (customizable) in
+    their config representation; 
+  - New `CoproductHint` trait which allows customization of the derived `ConfigConvert` for sealed families;
+- Breaking changes
+  - The default config representation for sealed families has changed. Configs for such classes that were read
```

Comment:
Yes, I agree with providing more information. In a bigger project there would be a section for logging changes and a separate migration guide; in this case, I don't think there is the need for that.